### ISSUE TITLE
ons-lazy-repeat 

### ISSUE THREAD

**Issue by d3orn** [OnsenUI/OnsenUI#1112]
It would be nice to have ons-lazy-repeat for other elements. For example with ng-repeat you can repeat any div or paragraph this should be possible with ons-lazy-repeat as well because it would actually improve ng-repeat.

**Comment by frankdiox** [comment]
@d3orn Oh, but Lazy Repeat in Onsen UI 1.x already works in any div: http://onsen.io/guide/overview.html#LazyRepeatInHTML

Since in Onsen UI 2.0 we won't have Angular directives or "CustomAttributes" we need to implement Lazy Repeat as a CustomElement (as happens with ons-ripple). Probably it will look something like this:

```
<ons-lazy-repeat ... >
   <div> ... </div>
</ons-lazy-repeat>
```

**Comment by IliaSky** [comment]
It's already implemented in 2.0 exactly as @frankdiox described it.
